js/private/js_image_layer.bzl

```diff
@@ -0,0 +1,140 @@
+"creates tar layers from js_binary targets"
+
+load("@aspect_bazel_lib//lib:paths.bzl", "to_manifest_path")
+load("@bazel_skylib//lib:paths.bzl", "paths")
+
+_doc = """Create container image layers from js_binary targets.
+
+```starlark
```

Comment:
missing some text here like "for example, you could..."

and/or link to the e2e where it's easier for us to show long starlark code listings